packages/lesswrong/server/search/utils.ts

```diff
@@ -230,7 +243,7 @@ Posts.toAlgolia = async (post: DbPost): Promise<Array<AlgoliaPost>|null> => {
       body: ""
     }));
   }
-  return postBatch;
+  return postBatch.reverse()
```

Comment:
I'm not sure about this, but I'm guessing that when Algolia deduplicates post objects, it uses the last one that was added. This causes the body snippet text to come from the end of the post rather than the start of the post. This is my attempt to fix that.

Reply:
worth a try!